package.json

```diff
@@ -39,6 +39,7 @@
     "github": "https://github.com/canjs/canjs"
   },
   "dependencies": {
+    "bit-docs-glob-finder": "0.0.6",
```

Comment:
I would like to better understand what changed that required this to be included.

[FYI, as of npm 5, any install will `--save` by default.]